### ISSUE TITLE
With twist.moe as the provider, DL command fails to decode data.

### ISSUE THREAD

**Issue by Zeexel** [anime-dl/anime-downloader#607]
**Describe the bug**

When the download command's provider variable is set to twist.moe, downloading will bring up an error regarding a decoding error.

**To reproduce**

```
1) Set the "provider" variable of the dl command to twist.moe
2) anime dl [anime name]
```

<details>
<summary>LOG</summary>

```
zeexel@regu:~$ anime dl 'nichijou
> '
anime: anime-downloader 5.0.5
+--------+----------------------------------------+--------+
|      5 | Danshi Koukousei no Nichijou           |        |
|      4 | Inou Battle wa Nichijou-kei no Naka de |        |
|      3 | Monster Musume no Iru Nichijou         |        |
|      2 | C³                                     |        |
|      1 | Nichijou                               |        |
|--------+----------------------------------------+--------|
|   SlNo | Title                                  | Meta   |
+--------+----------------------------------------+--------+
Enter the anime no: [1]: 1
anime: Selected Nichijou
anime: Extracting episode info from page
ERROR: 'utf-8' codec can't decode byte 0x9f in position 3: invalid start byte
```
![image](https://user-images.githubusercontent.com/41706751/104831567-69730500-5847-11eb-8cff-de0670c1d596.png)

**Comment by IguanasInPyjamas** [comment]
Yes, twist.moe changed server, cdns and keys to the videos. We are working on a fix right now. 
We have it downloading again from twist but not streaming. Hold on a lil longer.

**Comment by Zeexel** [comment]
Was fixed by #609
